walnut/README.md

```diff
@@ -1,4 +1,76 @@
 # `walnut` - the O² workflow administration and linting utility
 
 The O² **w**orkflow **a**dministration and **l**i**n**ting **ut**ility is a command line program for workflow configuration
-and import validation.
\ No newline at end of file
+and import validation.
+
+## Configuration file
+
+You can check the local `walnut` configuration with
+```bash
+$ walnut about
+```
+
+## Using `walnut`
+
+At any step, you can type `$ walnut help <subcommand>` to get information on what you can do and how, for example:
+
+```bash
+$ walnut help 
+walnut is a command line program for interacting with the O² Workflow Administration and Linting Utility.
+
+For more information on the available commands, see the individual documentation for each command.
+
+Usage:
+  walnut [command]
+
+Available Commands:
+  check       check the file passed against a specified schema.
+  convert     Converts a DPL Dump to the required formats.
+  help        Help about any command
+
+Flags:
+      --config string   optional configuration file for walnut (default $HOME/.config/walnut/settings.yaml)
+  -h, --help            help for walnut
+  -t, --toggle          Help message for toggle
```

Comment:
What's toggle?

Reply:
Not sure, must've been something added automatically by `cobra`. Removed.